crates/bdk/src/wallet/mod.rs

```diff
@@ -1350,7 +1352,7 @@ impl<D> Wallet<D> {
             utxos: original_utxos,
             bumping_fee: Some(tx_builder::PreviousFee {
                 absolute: fee,
-                rate: fee_rate.as_sat_per_vb(),
+                rate: fee_rate.to_sat_per_vb_ceil() as f32,
             }),
```

Comment:
this can be `rate: fee_rate` if the type of `rate` in `PreviousFee` is changed to `bitcoin::FeeRate`